src/meltano/cli/schedule.py

```diff
@@ -350,13 +356,33 @@ def _update_elt_schedule(
     return candidate
 
 
+class CronParam(click.ParamType):
+    """Custom type definition for cron prameter."""
+
+    name = "cron"
+
+    def convert(self, value, *_):
+        """Validate and con interval."""
+        if value is None:
+            return None
+
```

Comment:
I think this code is unreachable, i.e. if `--interval` is not passed to the command, this method will not even be called and if `--interval` is indeed passed, then `value` is not `None`.

It can probably be removed:

```suggestion
```

Reply:
Make sense